packages/nextjs/test/serverIntegrations.test.ts

```diff
@@ -0,0 +1,49 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+import { defaultRewriteFrames, getFinalServerIntegrations, IntegrationFunction } from '../src/utils/serverIntegrations';
+
+describe('user integrations without RewriteFrames', () => {
+  test('as an array', () => {
+    const userIntegrations: Integration[] = [];
+    // Should get a single integration: RewriteFrames
+    let finalIntegrations = getFinalServerIntegrations(userIntegrations);
+    expect(Array.isArray(finalIntegrations)).toBeTruthy();
```

Comment:
```suggestion
    expect(finalIntegrations).toBeInstanceOf(Array);
```

https://jestjs.io/docs/expect#tobeinstanceofclass

If you use more specific APIs, the error message when the test fails is considerably more useful.